src/style.css

```diff
@@ -0,0 +1,37 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 96px;
+  height: 16px;
```

Comment:
Don't set the container size, set size for individual items and let them dictate size for container.
Check the sizes and margin value on the mockup.
This will help with tests.